### ISSUE TITLE
Check if docs-screenshot and a11y tests are working on CI and locally

### ISSUE THREAD

**Issue by vitorwtavares** [camunda/camunda#26681]
### Description (required on creation)

A while back we had a PR that introduced a new URL parth, adding `/operate` to it, from that there were several problems that happened with tests, mostly E2E and visual regression. That situation was fixed whenever a situation happened but the fixes were done for specific types of tests, that means a11y and docs-screenshots tests have not been checked (and fixed, if needed) and therefore might not be working properly locally or on the CI.

The goal for this issue is to run these test suites locally and on the CI to double check everything is running as it should, and if not, then fix the problems.

### Acceptance Criteria (required on creation)

- [x] a11y tests are working both on CI and local environments or if they're failing, create an issue to fix them
- [x] docs-screenshots tests are working both on CI and local environments or if they're failing, create an issue to fix them

### Links

related to https://github.com/camunda/camunda/pull/18604 (introduction of `/operate` in base path)
related to https://github.com/camunda/camunda/pull/24241 (fix for E2E tests that were broken locally)

```[tasklist]
### Pull Requests
```

**Comment by vitorwtavares** [comment]
Created another issue to actually fix both test suites - https://github.com/camunda/camunda/issues/26977
